release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -73,6 +73,10 @@ for details.
 [id="ocp-4-4-security"]
 === Security
 
+==== The `oauth-proxy` imagestream is now available
```

Comment:
The text looks good, but is this better suited for the deprecated features section?

Reply:
@sheriff-rh So it's not technically deprecated or anything, we just want them to use the imagestream instead of the image. wdyt?

Reply:
Ah okay, that makes sense then - LGTM!